README.md

```diff
@@ -165,6 +166,31 @@ TODO: Write good description for anyone who wants to customize behavior
 1. Different strategies for failure predictor
 1. Integration for git hook
 
+## RSpec Runner
+
+There is a custom RSpec runner you can use in your development. It builds a prediction and runs it.
+
+### Configuration
+
+#### Config file
+
+Create a YAML file for the runner. Default location is `./config/crystalball.yml` or simply `./crystalball.yml`. Or you can create it in another place but define `CRYSTALBALL_CONFIG` env variable with a path to it.
+You can find an example of a config file in [`spec/fixtures/crystalball.yml`](https://github.com/toptal/crystalball/blob/master/spec/fixtures/crystalball.yml)
+
+#### Environment variables
+
+Set `CRYSTALBALL_SKIP_MAP_CHECK` to `true` if you want to receive a warning about an outdated map
+Set `CRYSTALBALL_SKIP_EXAMPLES_LIMIT` to a desired amount of examples which can be run automatically. It is not recommended to leave this parameter unconfigured or to set it to an unreasonably high number.
```

Comment:
This explanation is wrong. this param is also boolean and allows to skip examples limit check completely.